recipes/imagl/all/conandata.yml

```diff
@@ -5,3 +5,12 @@ sources:
   "0.1.1":
     url: "https://gitlab-lepuy.iut-clermont.uca.fr/opengl/imagl/-/archive/v0.1.1/imagl-v0.1.1.tar.gz"
     sha256: "4a7502cc733431af6423246fe5144e2eddb984454a66cca51742c852980ac862"
+  "0.1.2":
+    url: "https://gitlab-lepuy.iut-clermont.uca.fr/opengl/imagl/-/archive/v0.1.2/imagl-v0.1.2.tar.gz"
+    sha256: "d1edf74e00f969a47dc56e4400b805600d6997270339d49e91b7c6112a2cb37e"
+  "0.2.0":
+    url: "https://gitlab-lepuy.iut-clermont.uca.fr/opengl/imagl/-/archive/v0.2.0/imagl-v0.2.0.tar.gz"
+    sha256: "12b165a7f94a970d01b312b1d08be65f10665d60794daf32fd38024f6edd34cc"
```

Comment:
Please remove this version since you've added a newer one ... and please update title please 🙏

Reply:
Ok, I won't release 0.2.0 through conan